grizli/prep.py

```diff
@@ -350,7 +384,7 @@ def apply_persistence_mask(flt_file, path='../Persistence', dq_value=1024,
     flt = pyfits.open(flt_file, mode='update')
 
     pers_file = os.path.join(path,
-             os.path.basename(flt_file).replace('_flt.fits', '_persist.fits'))
+             os.path.basename(flt_file).replace('_flt.fits', '_persist.fits').replace('_rate.fits', '_persist.fits'))
```

Comment:
this is probably OK, but we don't know what persistence products might (eventually) be available for JWST